bridgesync/processor.go

```diff
@@ -59,6 +59,25 @@ const (
 	// orderByBlockDesc is the default order by clause for block-based queries
 	orderByBlockDesc = "block_num DESC, block_pos DESC"
 
+	// claimColumnsSQL is the list of all claim columns
+	claimColumnsSQL = `block_num,
+		block_pos,
+		tx_hash,
+		global_index,
+		origin_network,
+		origin_address,
+		destination_address,
+		amount,
+		proof_local_exit_root,
+		proof_rollup_exit_root,
+		mainnet_exit_root,
+		rollup_exit_root,
+		global_exit_root,
+		destination_network,
+		metadata,
+		is_message,
+		block_timestamp`
```

Comment:
Is it doable to craft some unit test that would use the reflection to check if all the fields in the `Claim` object, annotated by `meddler` tag, are enumerated here? Also it would imply to change the field to string slice (so it is easier to manipulate in the test), and where necessary to use it, when building the query and injecting args to query, use `strings.Join(...)`. WDYT?